flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -143,6 +153,23 @@ private Operation convertDropTable(SqlDropTable sqlDropTable) {
 		return new DropTableOperation(sqlDropTable.fullTableName(), sqlDropTable.getIfExists());
 	}
 
+	/** Convert CREATE FUNCTION statement. */
+	private Operation convertCreateFunction(SqlCreateFunction sqlCreateFunction) {
+		CatalogFunction catalogFunction =
+			new CatalogFunctionImpl(
+				sqlCreateFunction.getFunctionClassName().toValue(),
+				new HashMap<String, String>());
```

Comment:
Is there anything we need to set here for the properties on catalog function? @bowenli86? should we create a new constructor in `CatalogFunctionImpl` that have default properties value? 
this seems to me an unnecessary api exposure and we might have to maintain these dangling default values in lots of places.